rust-toolchain.toml

```diff
@@ -1,6 +1,6 @@
 # One should also update osdk/src/commands/new/rust-toolchain.toml.template
 # when updating this
 [toolchain]
-channel = "nightly-2024-11-29"
+channel = "nightly-2025-02-01"
```

Comment:
Did you try out the new Rust toolchain? Every thing compiles and every test passes?